src/frontend/src/components/public/PIAFormTabs/PIAStoredPersonalInformation/index.tsx

```diff
@@ -0,0 +1,770 @@
+import MDEditor from '@uiw/react-md-editor';
+import { useState } from 'react';
+import { useNavigate, useOutletContext } from 'react-router-dom';
+import { PiaStateChangeHandlerType } from '../../../../pages/PIAIntakeForm';
+import { YesNoInput } from '../../../../types/enums/yes-no.enum';
+import { IPiaForm } from '../../../../types/interfaces/pia-form.interface';
+import { isMPORole } from '../../../../utils/helper.util';
+import List, { InputTextProps } from '../../../common/List';
+import Radio from '../../../common/Radio';
+import {
+  IStoredPersonalInformation,
+  PrivacyRisk,
+  ServiceProviderDetails,
+} from './interfaces';
+import Messages from './messages';
+
+const StoredPersonalInformation = () => {
+  // TODO: Add prop for making everything in readonly for the view version of the tab
```

Comment:
We can create a separate component for readOnly version of this component. And redirect via Router.